skops/card/_model_card.py

```diff
@@ -165,6 +198,100 @@ def metadata_from_config(config_path: Union[str, Path]) -> ModelCardData:
     return card_data
 
 
+def split_subsection_names(key: str) -> list[str]:
+    r"""Split a string containing multiple sections into a list of strings for
+    each.
+
+    The separator is ``"/"``. To avoid splitting on ``"/"``, escape it using
+    ``"\\/"``.
+
+    Examples
+    --------
+    >>> split_subsection_names("Section A")
+    ['Section A']
+    >>> split_subsection_names("Section A/Section B/Section C")
+    ['Section A', 'Section B', 'Section C']
+    >>> split_subsection_names("A section containg \\/ a slash")
+    ['A section containg / a slash']
+    >>> split_subsection_names("Spaces are / stripped")
+    ['Spaces are', 'stripped']
+
+    Parameters
+    ----------
+    key : str
+        The section name consisting potentially of multiple subsections. It has
+        to be ensured beforhand that this is not an empty string.
+
+    Returns
+    -------
+    parts : list of str
+        The individual (sub)sections.
+
+    """
+    placeholder = "$%!?"  # arbitrary sting that never appears naturally
+    key = key.replace("\\/", placeholder)
+    parts = (part.strip() for part in key.split("/"))
+    return [part.replace(placeholder, "/") for part in parts]
+
+
+def _getting_started_code(
+    file_name: str, is_skops_format: bool = False, indent="    "
+) -> list[str]:
+    # get lines of code required to load the model
+    lines: list[str] = []
+    if is_skops_format:
+        lines += ["from skops.io import load"]
+    else:
+        lines += ["import joblib"]
+
+    lines += [
+        "import json",
+        "import pandas as pd",
+    ]
+    if is_skops_format:
+        lines += [
+            "from skops.io import load",
+            f'model = load("{file_name}")',
+        ]
+    else:  # pickle
+        lines += [f'model = joblib.load("{file_name}")']
```

Comment:
the consensus from discord seems to be `import skops.io as sio`

Reply:
Oh well, democracy has spoken. I changed it here, but would prefer to change the other occurrences (docs, examples) for another PR, since this one is already big as is.